browser/src/core/Socket.js

```diff
@@ -765,7 +765,7 @@ app.definitions.Socket = L.Class.extend({
 			if (parseInt(h,16).toString(16) === h.toLowerCase().replace(/^0+/, '')) {
 				const anchor = document.createElement('a');
 				anchor.setAttribute('target', '_blank');
-				anchor.setAttribute('href', 'https://hub.libreoffice.org/git-core/' + h);
+				anchor.setAttribute('href', 'https://git.libreoffice.org/core/+log/' + lokitVersionObj.BuildId + '/');
```

Comment:
Ah, the commit message explains what I was wondering at https://gerrit.libreoffice.org/c/core/+/184820/2#message-ae4c0269d4de8ccce697e3a0d2d56917937005e4